pkg/webui/lib/store/middleware/collaborators.js

```diff
@@ -73,11 +73,8 @@ export default tts => {
   const deleteCollaboratorLogic = createRequestLogic({
     type: collaborators.DELETE_COLLABORATOR,
     process: async ({ action }) => {
-      const { parentType, parentId } = action.payload
-
-      const result = await tts.Clients.Collaborators.update(parentType, parentId)
-
-      return result
+      const { parentType, parentId, collaboratorId, isUser } = action.payload
+      return tts[entitySdkServiceMap[parentType]].Collaborators.remove(isUser, parentId, collaboratorId)
```

Comment:
```suggestion
      const result = await tts[entitySdkServiceMap[parentType]].Collaborators.remove(isUser, parentId, collaboratorId)
      
      return result
```

The reason to add this extra step is to ensure that errors during fetching are thrown in the correct scope.